src/platform/packages/shared/response-ops/alerts-apis/hooks/use_get_muted_alerts_query.tsx

```diff
@@ -38,13 +38,15 @@ export interface UseGetMutedAlertsQueryParams {
   notifications: NotificationsStart;
 }
 
+export const getKey = queryKeys.getMutedAlerts;
```

Comment:
Trying the WIP proposal of co-locating key creators by re-exporting them from the factory

Reply:
what's the idea behind this?